src/main/java/com/csselect/database/mysql/MysqlGameAdapter.java

```diff
@@ -0,0 +1,325 @@
+        return players;
+    }
+
+    @Override
+    public Collection<Round> getRounds() {
+        Collection<Round> rounds = new HashSet<>();
+        try {
+            ResultSet set = DATABASE_ADAPTER.executeMysqlQuery("SELECT * FROM rounds", databaseName);
+            while (set.next()) {
+                rounds.add(getGamemode().createRound(new Player(new MysqlPlayerAdapter(
+                        set.getInt("player_id"))), set.getInt("id")));
+            }
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+        return rounds;
+    }
+
+    @Override
+    public Gamemode getGamemode() {
+        String gamemode = getString("gamemode");
+        if (gamemode.startsWith("BinarySelect")) {
+            return new BinarySelect();
+        } else {
+            String[] args = gamemode.split(",");
+            return new MatrixSelect(Integer.parseInt(args[1]), Integer.parseInt(args[2]), Integer.parseInt(args[3]));
+        }
+    }
+
+    @Override
+    public Termination getTermination() {
+        String[] terminations = getString("termination").split(",");
+        TerminationComposite termination = new TerminationComposite();
+        for (String t : terminations) {
+            if (t.startsWith("time")) {
+                termination.add(new TimeTermination(LocalDateTime.ofEpochSecond(
+                        Long.parseLong(t.replace("time:", "")), 0, ZoneOffset.UTC)));
+            } else if (t.startsWith("rounds")) {
+                termination.add(
+                        new NumberOfRoundsTermination(Integer.parseInt(t.replace("rounds:", ""))));
+            }
+        }
+        return termination;
+    }
+
+    @Override
+    public Organiser getOrganiser() {
+        return new Organiser(new MysqlOrganiserAdapter(getInt("organiser_id")));
+    }
+
+    @Override
+    public void setTermination(Termination termination) {
+        setString("termination", termination.toString());
+
+    }
+
+    @Override
+    public void setFeatures(FeatureSet featureSet) {
+        setString("dataset", featureSet.getIdentifier());
+    }
+
+    @Override
+    public void setFinished() {
+        setBoolean("isTerminated", true);
+    }
+
+    @Override
+    public void setTitle(String title) {
+        setString("title", title);
+    }
+
+    @Override
+    public void setDescription(String description) {
+        setString("description", description);
+    }
+
+    @Override
+    public void setDatabase(String name) {
+        databaseName = name;
+        setString("databaseName", name);
+        try {
+            createPlayersTable();
+            createRoundsTable();
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+    }
+
+    @Override
+    public void setGamemode(Gamemode gamemode) {
+        setString("gamemode", gamemode.toString());
+    }
+
+    @Override
+    public void setOrganiser(Organiser organiser) {
+        setInt("organiser_id", organiser.getId());
```

Comment:
change organiser_id to organiserId, because in other locations the names are also in camelCase not in snake_case